lib/barbeque/message.rb

```diff
@@ -9,10 +9,14 @@ module Message
     class << self
       # @param [Aws::SQS::Types::Message] raw_message
       # @return [Barbeque::Message::Base]
-      def parse(raw_message)
+      def parse(raw_message, job_queue:)
```

Comment:
Only one argument lacks type notation. I think writing all or nothing is normal.